client/src/components/Overview/OverviewResults.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import PropTypes from "prop-types";
```

Comment:
Do you plan on this the propTypes? I see it's commented out. If we want to use them I'd say let's use them otherwise let's take the dependency out. (I'm indifferent either way. Just don't want excessive dependencies. )

This comment goes for any file that has prop-types as a dependency.

Reply:
Yes, I do prefer to use proptypes, but I haven't implemented them yet